app/code/local/Magmodules/Webwinkelconnect/Model/Api.php

```diff
@@ -77,52 +77,47 @@ public function sendInvitation($order)
     {
         $startTime = microtime(true);
         $crontype = 'orderupdate';
-        $orderId = $order->getIncrementId();
         $storeId = $order->getStoreId();
         $apiId = trim(Mage::getStoreConfig('webwinkelconnect/general/api_id', $storeId));
         $apiKey = trim(Mage::getStoreConfig('webwinkelconnect/general/api_key', $storeId));
-        $delay = trim(Mage::getStoreConfig('webwinkelconnect/invitation/delay', $storeId));
-        $language = Mage::getStoreConfig('webwinkelconnect/invitation/language', $storeId);
-        $resendDouble = Mage::getStoreConfig('webwinkelconnect/invitation/resend_double', $storeId);
-
-        $email = $order->getCustomerEmail();
-        $customerName = $order->getCustomerName();
-
-        $url = 'https://www.webwinkelkeur.nl/api.php?id=' . $apiId;
-        $url .= '&password=' . $apiKey . '&email=' . urlencode($email) . '&order=' . $orderId;
-        $url .= '&delay=' . $delay . '&client=magento&customername=' . urlencode($customerName);
-
-        if (!$resendDouble) {
-            $url = $url . '&noremail=1';
-        }
-
-        if (!empty($language)) {
-            if ($language == 'cus') {
-                $lanArray = array('NL' => 'nld', 'EN' => 'eng', 'DE' => 'deu', 'FR' => 'fra', 'ES' => 'spa');
-                $address = $order->getShippingAddress();
-                if (isset($lanArray[$address->getCountry()])) {
-                    $url = $url . '&language=' . $lanArray[$address->getCountry()];
-                }
-            } else {
-                $url = $url . '&language=' . $language;
-            }
+        $postData['email'] = $order->getCustomerEmail();
+        $postData['order'] = $order->getIncrementId();
+        $postData['delay'] = trim(Mage::getStoreConfig('webwinkelconnect/invitation/delay', $storeId));
+        $postData['customer_name'] = $order->getCustomerName();
+        $postData['client'] = 'magento';
```

Comment:
```suggestion
        $postData['client'] = 'magento1';
```

It would be also nice to add the version. But not really a requirement.